package/crds/rds.aws.crossplane.io_dbclusters.yaml

```diff
@@ -1090,7 +1090,6 @@ spec:
                     type: array
                 required:
                 - engine
-                - masterUserPasswordSecretRef
```

Comment:
This change here is going to be overwritten by kubebuilder. You have to set the json property to `omitempty` here: https://github.com/crossplane-contrib/provider-aws/blob/f0163fa10ab2856dcc9429e0c123d70b0c863e20/apis/rds/v1alpha1/custom_types.go#L197

Reply:
Thanks for pointing me in the right direction!